integration/spark/src/main/antlr4/org/apache/spark/sql/parser/CarbonSqlBase.g4

```diff
@@ -0,0 +1,1842 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
I see that 2 .g4 files created. This is required ? I think it is generated file

Reply:
@ajantha-bhat only 1 g4 file is there, this g4 file is used to gen the other java files, and the other java files will only generated during compile process.